src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/actions.py

```diff
@@ -5,6 +5,8 @@
 
 import argparse
 
+import antlr4
```

Comment:
Was it brought in by the monitor module?

Reply:
Yes. This is needed for the grammar parser.